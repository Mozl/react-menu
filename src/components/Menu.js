import React from 'react';
import styled from 'styled-components';
import Starters from './Starters';
import menuData from '../menu-data';

const MenuWrapper = styled.div`
  display: flex;
  border: 0.1rem solid black;
  flex-direction: column;
  text-color: black;
  border-radius: 1.25rem;
  padding: 1rem;
  margin-top: 1rem;
`;

MenuWrapper.displayName = 'MenuWrapper';

class Menu extends React.Component {
  render() {
    return (
      <MenuWrapper>
        <h1>Menu</h1>
        <Starters dishes={menuData} />
      </MenuWrapper>
    );
  }
}
export default Menu;
