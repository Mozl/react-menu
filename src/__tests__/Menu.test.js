import React from 'react';
import ReactDOM from 'react-dom';
import Menu from '../components/Menu';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Menu />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders correctly', () => {
  const tree = renderer.create(<Menu />);
  expect(tree).toMatchSnapshot();
});
