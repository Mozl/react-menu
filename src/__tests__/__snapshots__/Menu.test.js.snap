// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-htpNat fLtbxA"
>
  <h1>
    Menu
  </h1>
  <div
    className="sc-bwzfXH gokkPN"
  >
    <h2>
      Starters
    </h2>
    <div
      className="sc-bdVaJa fVezFz"
    >
      Soup
    </div>
    <div
      className="sc-bdVaJa fVezFz"
    >
      Pâté
    </div>
    <div
      className="sc-bdVaJa fVezFz"
    >
      Bruschetta
    </div>
    <div
      className="sc-bdVaJa fVezFz"
    >
      Prawn cocktail
    </div>
  </div>
</div>
`;
